.editorconfig

```diff
@@ -14,6 +14,87 @@ indent_size = 4
 [project.json]
 indent_size = 2
 
+# C# files
+[*.cs]
+# brace begins on a new line
+csharp_new_line_before_open_brace = all
+
+# block indentation
+csharp_indent_block_contents = true
+
+# avoid this. unless absolutely necessary
+dotnet_style_qualification_for_field = false:suggestion
+dotnet_style_qualification_for_property = false:suggestion
+dotnet_style_qualification_for_method = false:suggestion
+dotnet_style_qualification_for_event = false:suggestion
+
+# only use var when it's obvious what the variable type is
+csharp_style_var_for_built_in_types = false:none
+csharp_style_var_when_type_is_apparent = false:none
```

Comment:
@mellinoe, the code-style documentation isn't explicit, but I believe the convention is that `csharp_style_var_for_built_in_types`, `csharp_style_var_when_type_is_apparent`, and `csharp_style_var_for_locals` should all be `true:suggestion`, correct?